### PR TITLE
init reactor greenlet earlier

### PR DESCRIPTION
**Pull request by ybilopolov**
implement #12

### CODE REVIEW

File: testing/test_basic.py [hunk 1]
```diff
@@ -103,3 +103,26 @@ def test_MAIN():
     rr = testdir.run(sys.executable, "-m", "pytest", "-v")
     outcomes = rr.parseoutcomes()
     assert outcomes.get("passed") == 1
+
+
+def test_blocon_in_hook(testdir):
+    testdir.makeconftest("""
+import pytest
```
Comment (nicoddemus) [inline testing/test_basic.py]: You can indent this as you will, `makeconftest` ensures the indent is correct:

```python
def test_blocon_in_hook(testdir):
    testdir.makeconftest("""
        import pytest
        from twisted.internet import reactor, defer
        def pytest_configure(config):
            d = defer.Deferred()
            reactor.callLater(0.01, d.callback, 1)
            pytest.blockon(d)
    """)
    testdir.makepyfile("""
        from twisted.internet import reactor, defer
```

Comment (ybilopolov) [reply testing/test_basic.py]: Oh, thanks, was just using the original file style. I've updated the indentation of other tests too.

Comment (nicoddemus) [reply testing/test_basic.py]: Cool thanks!

**Comment by ybilopolov** [comment]
With this, I believe there is no need to have `init_twisted_greenlet` in pytest namespace anymore (which was added in #10 to solve similar issue), updated the PR.

**Comment by vtitor** [comment]
@ybilopolov Thanks for the great job.

**Pull request merged**